_pages/install/integrations/opf-integration/opf-child-pages/1-open-payment-framework-in-spartacus.md

```diff
@@ -0,0 +1,267 @@
+---
+title: Open Payment Framework in Spartacus
+---
+
+The SAP Commerce Open Payment Framework, which is delivered as an enrichment to the SAP Commerce Cloud payment toolkit, is a SaaS solution for managing your payment integrations in an intuitive and effective way. The Open Payment Framework allows you to integrate your preferred digital payment service providers faster than before, and removes the need to code, integrate and deploy extensions to the Commerce codebase.
+
+## Enabling Open Payment Framework in Spartacus
+
+To enable open payment framework, install the `@spartacus/opf` integration library (???). For more information, see [Installing Additional Composable Storefront Libraries](link).
+
```

Comment:
@FollowTheFlo @Matejk00 "To enable open payment framework, install the `@spartacus/opf` integration library (???)."
Actually, I just made this up when I was setting up the docs. Is this actually true? Is this the correct name for the lib? 😄